src/var/isFunction.js

```diff
@@ -3,12 +3,14 @@ define( function() {
 
 	return function isFunction( obj ) {
 
-		// Support: Chrome <=57, Firefox <=52
+		// Support: Chrome <=57, Firefox <=52, QtWeb <=3.8.5, WebKit <=534.34,
+		// wkhtmltopdf tool <=0.12.5
```

Comment:
I would add a separate support comment below the existing one as the issue it covers is different. See an example here:
https://github.com/jquery/jquery/blob/fa0058af426c4e482059214c29c29f004254d9a1/src/event.js#L366-L370

Reply:
Agree. Checked in already.